tests/CMakeLists.txt

```diff
@@ -711,14 +711,17 @@ foreach(pol IN LISTS DSLASH_POLICIES)
         --gtest_output=xml:dslash_mobius_eofa_test_pol${pol2}.xml)
 
     # MdagM local operator
-    add_test(NAME dslash_mobius_local-policy${pol2}
-             COMMAND ${QUDA_CTEST_LAUNCH} $<TARGET_FILE:dslash_ctest> ${MPIEXEC_POSTFLAGS}
-                     --dslash-type mobius
-                     --test MatPCDagMatPCLocal
-                     --matpc even-even
-                     --dim 2 4 6 8
-                     --Lsdim 4
-                     --gtest_output=xml:dslash_mobius_local{pol2}.xml)
+    # This requires tensor cores so only enable NVIDIA targets
+    if( QUDA_TARGET_CUDA )
```

Comment:
Is there a version of this test that does not use tensor cores?

Reply:
@hummingtree ?

Reply:
No there is not a non-tensor-core version of this. Disabling this test is the right thing to do I believe. I wonder what would happen if the user calls this operator with a HIP build?

Reply:
Ah I see that `QUDA_MMA_AVAILABLE` is only set when the target is CUDA.